cluster/gce/config-default.sh

```diff
@@ -96,6 +96,7 @@ NETWORK_POLICY_PROVIDER="${NETWORK_POLICY_PROVIDER:-flannel}" # calico kubenet d
 if [[ "${NETWORK_PROVIDER:-}" == "mizar" ]]; then
   NETWORK_PROVIDER_VERSION="dev"
   NETWORK_POLICY_PROVIDER="mizar"
+  CNI_BIN_DIR="/opt/cni/bin"
```

Comment:
Please add the following statement in the if block for test to pass in kube-up:
KUBE_CONTAINER_RUNTIME="docker"

Reply:
added.